docs/node-configuration.md

```diff
@@ -158,6 +159,10 @@ where:
 - `MaxNEP11Tokens` - limit for the number of tokens returned from
   `getnep11balances` call.
 - `Port` is an RPC server port it should be bound to.
+- `StartWhenSynchronized` controls when RPC server will be started, by default
+  (`false` setting) it's started immediately and RPC is availabe during node
+  synchronization. Setting it to `true` will make the node start RPC service only
+  after full synchronization.
```

Comment:
Probably, it should be mentioned here that RPC service will be started after the node reaches 2/3 of the connected peers height.

Reply:
It affects all of the services, so I've added another section.